src/kili/services/data_connection/__init__.py

```diff
@@ -160,7 +190,7 @@ def synchronize_data_connection(
             )
 
     if added > 0:
-        data_connection = validate_data_differences(auth, "ADD", data_connection_id)
+        validate_data_differences(auth, "ADD", data_connection)
```

Comment:
ADD and REMOVED are necessarily done in two different calls ?

Reply:
yes I think:

```graphql
  validateDataDifferences(where: ValidateDataDifferencesWhere!, processingParameters: String): DataConnection

input ValidateDataDifferencesWhere {
  connectionId: ID!
  type: DataDifferenceType!
}

enum DataDifferenceType {
  ADD
  REMOVE
}
```

Reply:
indeed!